HousingSearchListener/V1/UseCase/Interfaces/IUpdateAccountDetailsUseCase.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace HousingSearchListener.V1.UseCase.Interfaces
+{
+    public interface IUpdateAccountDetailsUseCase : IMessageProcessing
+    {
```

Comment:
We need to add the methods here, not sure how your test is working with this empty